jupyter/share/.jupyter/jupyter_notebook_config.py

```diff
@@ -7,8 +7,6 @@
 # Allow requests from the proxy
 c.NotebookApp.allow_origin = '*'
 
-c.NotebookApp.nbserver_extensions = {
-    'jupyter_localize_extension': True,
-}
+c.NotebookApp.nbserver_extensions = {}
```

Comment:
`jupyter_localize_extension` does not work in existing state. It fails to install while launching the notebook with error
```
    ModuleNotFoundError: No module named 'jupyter_localize_extension'
```

Reply:
thanks for taking care of this